### PR TITLE
Has Left room was showing two times and has been fixed

### LINKED ISSUE #171: "Has left the room" message displayed twice
When the user OneUser quits, the other participants see the message "OneUser has left the room" twice. That should not be the case.

### PR DESCRIPTION
**Pull request by mohsini172**
#171 When a user leaves the room, the action service was calling the destroy feed two times one for leaving and one for unPublish.

### CODE REVIEW

**Comment by mohsini172** [comment]
@imobachgs please review changes

**Comment by ancorgs** [comment]
Not sure if adding a parameter to `ActionService.destroyFeed` is the right thing to do here. I would say we actually need two separate actions. One for unpublishing and one for leaving.

**Comment by mohsini172** [comment]
Ok I will check the code if both of them can be separated.

**Comment by imobachgs** [comment]
Thanks for your contribution @mohsini172.

Are you still interested in fixing this issue? Maybe you would like to cooperate with @aniabui in order to fix it. If you're not interested, just close the PR.

Thanks in advance!

**Comment by mohsini172** [comment]
Yeah I will fix it within two days.

**Comment by mohsini172** [comment]
@imobachgs Please review changes now.

File: src/app/components/room/action.service.js [hunk 1]
```diff
@@ -83,6 +84,15 @@
       LogService.add(entry);
```
Comment (ancorgs) [inline src/app/components/room/action.service.js]: I would go one step further and re-implement `destroyFeed`to look like that:
```javascript
function destroyFeed(feedId) {
       unpublishFeed(feedId);
       // Log the event
       var entry = new LogEntry("destroyFeed", {feed: feed});
       LogService.add(entry);
}
```

That will reduce code duplication.

Comment (ancorgs) [inline src/app/components/room/action.service.js]: Oh, no. I was wrong. Never mind.

**Pull request merged**